src/components/Pollution.jsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import { faArrowLeft, faMicrophone, faGear} from "@fortawesome/free-solid-svg-icons";
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useLocation, useNavigate } from "react-router-dom";
+import { fetchPollutionData,selectPollutionData } from "../redux/pollutionSlice";
+
+const Pollution = () => {
+
+  let location = useLocation()
+  let data = location.state
+  let navigate = useNavigate()
+  const handleGoBackButton = ()=>{
+    navigate(-1)
+  }
+
+  let pollutionData = useSelector(selectPollutionData(data.name))
+  let dispatch = useDispatch()
+
+  useEffect(() => {
+    if(!pollutionData){
+
+      dispatch(fetchPollutionData({country:data.name,lat:data.lat,lon:data.lon}))
+    }
+
+
+  }, [data])
+
+
+  return (
+<>
+<div className = "pollution-card-header">
+<FontAwesomeIcon className = "left-arrow-icon" onClick={handleGoBackButton} icon={faArrowLeft} size="1xs" />
+<FontAwesomeIcon className = "microphone-icon" icon={faMicrophone} size="1xs" />
+<FontAwesomeIcon className = "gear-icon" icon={faGear} size="1xs" />
+</div>
+<div className="pollution-card">
+<div className="pollution-details">
+  <h1>{data.name}</h1>
+  <h2>Pollution Data</h2>
+  <p>Air Pollutants</p>
+  <p>no : {pollutionData.no}</p>
+  <p>co : {pollutionData.co}</p>
+  <p>no2 : {pollutionData.no2}</p>
```

Comment:
- [ ] When I navigate to the details page of the country I see only the following error please make sure to fix it and also it should be the same as the design in order to complete the requirements 
- Yours 
<img width="314" alt="Screenshot 2023-06-03 at 17 01 52" src="https://github.com/LDouglasOT/AirPollution/assets/61790587/4e1eee48-caea-47f3-8b67-069e9cf43090">

- Design 
<img width="257" alt="Screenshot 2023-06-03 at 16 59 57" src="https://github.com/LDouglasOT/AirPollution/assets/61790587/514db41c-fb1a-4afe-a2ec-1f8d0384d922">

Reply:
fixed